qemu/tests/vdi_image_convert.py

```diff
@@ -0,0 +1,33 @@
+from avocado import fail_on
+from avocado.utils import process
+
+from virttest.qemu_io import QemuIOSystem
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    1. Convert images between raw and vdi.
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def _qemu_io(img, cmd):
+        """Run qemu-io cmd to a given img."""
+        try:
+            QemuIOSystem(test, params, img.image_filename).cmd_output(cmd, 120)
+        except process.CmdError:
+            test.fail("qemu-io to '%s' failed." % img.image_filename)
```

Comment:
test.error here? This is not a test scenario of this case.

Reply:
Updated, thanks.